control/controlclient/auto.go

```diff
@@ -59,15 +59,17 @@ type Auto struct {
 
 	mu sync.Mutex // mutex guards the following fields
 
-	paused          bool // whether we should stop making HTTP requests
-	unpauseWaiters  []chan struct{}
-	loggedIn        bool       // true if currently logged in
-	loginGoal       *LoginGoal // non-nil if some login activity is desired
-	synced          bool       // true if our netmap is up-to-date
-	inPollNetMap    bool       // true if currently running a PollNetMap
-	inLiteMapUpdate bool       // true if a lite (non-streaming) map request is outstanding
-	inSendStatus    int        // number of sendStatus calls currently in progress
-	state           State
+	paused               bool // whether we should stop making HTTP requests
+	unpauseWaiters       []chan struct{}
+	loggedIn             bool       // true if currently logged in
+	loginGoal            *LoginGoal // non-nil if some login activity is desired
+	synced               bool       // true if our netmap is up-to-date
+	inPollNetMap         bool       // true if currently running a PollNetMap
+	inLiteMapUpdate      bool       // true if a lite (non-streaming) map request is outstanding
+	liteMapUpdateCancel  func()     // cancels a lite map update
```

Comment:
can it be nil? why not an context.CancelFunc? If it's a CancelFunc, then I know it's harmless to call multiple times.

Reply:
updated docs.